examples/lib/stories/effects/dual_effect_removal_example.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flutter/material.dart';
+import '../../commons/ember.dart';
+
+class DualEffectRemovalExample extends FlameGame with TapDetector {
+  static const String description = '''
+    In this example we show how a dual effect can be used and removed.
+    To remove an effect just tap anywhere on the screen.
+    The 1st tap will remove the OpacityEffect and the 2nd tap removes the ColorEffect.
+  ''';
+
+  late final SpriteComponent mySprite;
+  late  ColorEffect fx;
+  late OpacityEffect opfx;

```

Comment:
how about calling these variables `colorEffect` and `opacityEffect`, so that it's clear which one's which.

Reply:
Couldn't this be fixed by having the effects doing this in onRemove? Feels strange that the user should have to do this.